bin/check_tron_jobs.py

```diff
@@ -231,15 +231,22 @@ def main():
     cmd_utils.load_config(args)
     client = Client(args.server)
 
+    error_code = 0
     if args.job is None:
         jobs = client.jobs(include_job_runs=True)
         for job in jobs:
-            check_job_result(job=job, client=client, dry_run=args.dry_run)
+            try:
+                check_job_result(job=job, client=client, dry_run=args.dry_run)
+            except Exception:
```

Comment:
except Exception as e 

and then log the output of e so we know "why" it failed